extras/nginx/nginx-1.16.patch

```diff
@@ -1806,6 +1806,7 @@ index 000000000..972ecd223
 +    ngx_http_v3_stream_t      *stream;
 +    ngx_http_v3_srv_conf_t    *h3scf;
 +    ngx_http_v3_connection_t  *h3c;
++    bool                       has_body;
```

Comment:
I think we need to initialize `has_body` to `false` (`http3-server.c` does that too)

Reply:
I think that doesn't suit nginx style (but I could be wrong). However after some offline discussion, we have a better API design that will avoid the problem. I'll push an update tomorrow.